Cargo.toml

```diff
@@ -13,14 +13,17 @@ readme = "README.md"
 documentation = "https://esp-rs.github.io/esp-idf-hal/"
 
 [features]
-default = ["std", "alloc", "esp-idf-sys", "embedded-svc"]
+default = ["std", "alloc", "esp-idf-sys", "embedded-svc", "pio"]
```

Comment:
We recently switched to `native` as the default for `esp-idf-sys`. The `pio` feature should be opt-in, not the default.